vignettes/meta_forestly.Rmd

```diff
@@ -0,0 +1,145 @@
+---
+title: "A closer look at meta_forestly()"
+output:
+  rmarkdown::html_document:
+    self_contained: no
+    number_sections: yes
+    code_folding: hide
+vignette: |
+  %\VignetteIndexEntry{Controlling Layout}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  message = FALSE,
+  warning = FALSE,
+  eval = TRUE,
+  echo = TRUE
+)
+```
+
+```{r}
+library(forestly)
+library(metalite)
+```
+
+# Overview
+
+`meta_forestly()` is the first step in the Interactive Forest Plot generation workflow and is a wrapper of several [metalite](https://merck.github.io/metalite/) functions. Specifically, it defines a metadata, including adam datesets, population, observations and parameters to be used in the forest plot. 
+In this [vignette](https://merck.github.io/forestly/articles/forestly.html#define-metadata) we will take a closer look at the `parameter_term` argument in `meta_forestly()` to better understand how to generate the forest plot for different parameters of interest.
+
+# `parameter_term` examples
+
+Prior to turning to the examples, just a reminder that the treatment group variable in the ADaM datasets should be set as a factor, otherwise a warning will be generated. 
+For example, if this is what you have in the `ADSL.TRT01A`:
```

Comment:
It's unclear what the `ADSL.TRT01A` format mean. The pandas way to mention a variable in a data frame?

Reply:
it is how you would expect it to appear in the spec, but I can remove ADSL part. It should be clear without it too